app/services/moderation.py

```diff
@@ -164,7 +164,7 @@ async def auto_restrict(target: User, chat: Chat, bot: Bot, using_db=None) -> ty
 async def get_count_auto_restrict(target: User, chat: Chat, bot_user: User = None, bot: Bot = None):
```

Comment:
This is a incorrect typehints.
```suggestion
async def get_count_auto_restrict(target: User, chat: Chat, bot_user: typing.Optional[User] = None, bot: typing.Optional[Bot] = None) -> int:
```

Reply:
fixed, thanks